src/SouffleInterface.h

```diff
@@ -984,3 +985,12 @@ class ProgramFactory {
     }
 };
 }  // namespace souffle
+
+#ifdef _MSC_VER
```

Comment:
Please add a comment explaining this code.

Reply:
This code was originally to fix a compiler error on MSVC, but I've removed it since something seems to have changed it now compiles fine without it.